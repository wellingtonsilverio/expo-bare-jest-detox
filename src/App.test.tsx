import { render, screen } from '@testing-library/react-native';

import App from './App';

describe('<App />', () => {
  it('should App snapshot', () => {
    render(<App />);

    expect(screen.toJSON()).toMatchSnapshot();
  });
});
