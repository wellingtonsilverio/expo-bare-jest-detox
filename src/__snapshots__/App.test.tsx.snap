// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should App snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    testID="welcome"
  >
    Open up App.js to start working on your app!
  </Text>
</View>
`;
